modules/tests/test/create_vm_common_test.go

```diff
@@ -162,144 +162,146 @@ var _ = Describe("Create VM", func() {
 					})
 				}
 			})
-		})
-		It("VM with attached PVCs/DVs and existing disks/volumes is created successfully", func() {
-			mode := "template-mode"
-			if c == CreateVMVMManifestMode {
-				mode = "manifest-mode"
-			}
-			datavolumes := []*datavolume.TestDataVolume{
-				datavolume.NewBlankDataVolume("attach-to-vm-with-disk-name-1-" + mode).AttachWithDiskName("disk1").AttachAs(datavolume.OwnedPVC),
-				datavolume.NewBlankDataVolume("attach-to-vm-with-disk-name-2-" + mode).AttachWithDiskName("disk2").AttachAs(datavolume.OwnedDV),
-				datavolume.NewBlankDataVolume("attach-to-vm-with-disk-name-3-" + mode).AttachWithDiskName("disk3").AttachAs(datavolume.OwnedDV),
-			}
-
-			vmDisk1 := kubevirtv1.Disk{
-				Name: datavolumes[0].DiskName,
-				DiskDevice: kubevirtv1.DiskDevice{
-					CDRom: &kubevirtv1.CDRomTarget{Bus: "sata"},
-				},
-			}
-			vmDisk2 := kubevirtv1.Disk{
-				Name: datavolumes[1].DiskName,
-				DiskDevice: kubevirtv1.DiskDevice{
-					Disk: &kubevirtv1.DiskTarget{Bus: "virtio"},
-				},
-			}
+			It("VM with attached PVCs/DVs and existing disks/volumes is created successfully, mode: "+string(c), func() {
```

Comment:
Use either `c` or `createMode`? Both are the same?

Reply:
fixed